sdk/client-core/core/src/main/java/io/clientcore/core/implementation/util/DefaultLogger.java

```diff
@@ -30,18 +28,14 @@ public final class DefaultLogger extends MarkerIgnoringBase {
     private static final String HYPHEN = " - ";
     private static final String OPEN_BRACKET = " [";
     private static final String CLOSE_BRACKET = "]";
-    public static final String WARN = "WARN";
-    public static final String DEBUG = "DEBUG";
-    public static final String INFO = "INFO";
-    public static final String ERROR = "ERROR";
-    public static final String TRACE = "TRACE";
+    private static final String WARN = "WARN";
+    private static final String DEBUG = "DEBUG";
+    private static final String INFO = "INFO";
+    private static final String ERROR = "ERROR";
+    private static final String NOTSET = "NOTSET";
```

Comment:
Any reason we changed `TRACE` to `NOTSET`? This seems separate from the SLF4J removal